code/datums/mob_stats.dm

```diff
@@ -14,11 +14,56 @@
 	var/datum/stat/S = stat_list[statName]
 	S.changeValue(Value)
 
-/datum/stat_holder/proc/getStat(statName, Pure = null)
-	var/datum/stat/S = stat_list[statName]
-	return S ? S.getValue(Pure) : 0
-
 
+/datum/stat_holder/proc/getStat(statName, Pure = null)
```

Comment:
Extended this proc to support compound stat checks

Reply:
May be that should be separated proc-wraper?

Reply:
No real need, this isn't performance critical code. This method is quite flexible